RHEL6: failing persistent overlay integration tests

### Version of Singularity:


```
3.5.2

```

### Actual behavior

RHEL6 does not support overlay - shouldn't be fatal here.

```
--- FAIL: TestSingularityActions/Persistent_Overlay (2.05s)
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_create (0.05s)
            actions_test.go:428: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:429: unexpected failure running 'test -f /dir_overlay'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_find (0.03s)
            actions_test.go:436: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:437: unexpected failure running 'test -f /dir_overlay'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_ext3_create (0.03s)
            actions_test.go:444: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:445: unexpected failure running 'test -f /ext3_overlay'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_ext3_find (0.03s)
            actions_test.go:452: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:453: unexpected failure running 'test -f /ext3_overlay'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_squashFS_find (0.03s)
            actions_test.go:460: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:461: unexpected failure running 'test -f /bogus863987367'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_multiple_create (0.03s)
            actions_test.go:468: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:469: unexpected failure running 'touch /multiple_overlay_fs'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_multiple_find_ext3 (0.03s)
            actions_test.go:476: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
            actions_test.go:477: unexpected failure running 'test -f multiple_overlay_fs'
        --- FAIL: TestSingularityActions/Persistent_Overlay/overlay_multiple_find_squashfs (0.03s)
            actions_test.go:483: FATAL:   overlay images requires overlay kernel support: your kernel doesn't support it
                 <nil>
```



<!-- if this is a feature request, you can ignore this next part -->

### Steps to reproduce this behavior

How can others reproduce this issue/problem?


### What OS/distro are you running

```
RHEL 6
```

--- a/cmd/singularity/actions_test.go
+++ b/cmd/singularity/actions_test.go
@@ -364,6 +364,7 @@ func testRunFromURI(t *testing.T) {
 
 // testPersistentOverlay test the --overlay function
 func testPersistentOverlay(t *testing.T) {
+	require.Filesystem(t, "overlay")
 	const squashfsImage = "squashfs.simg"
 	//  Create the overlay dir
 	cwd, err := os.Getwd()

